odoo/addons/base/tests/test_ir_attachment.py

```diff
@@ -225,6 +227,34 @@ def test_11_copy(self):
         self.assertEqual(document3.store_fname, self.blob1_fname)
         self.assertEqual(document3.checksum, self.blob1_hash)
 
+    def test_12_gc(self):
+        # the data needs to be unique so that no other attachment link
+        # the file so that the gc removes it
+        unique_blob = uuid.uuid4().bytes
+        a1 = self.Attachment.create({'name': 'a1', 'raw': unique_blob})
+        store_path = os.path.join(self.filestore, a1.store_fname)
+        self.assertTrue(os.path.isfile(store_path), 'file exists')
+        a1.unlink()
+        self.Attachment._gc_file_store_unsafe()
+        self.assertFalse(os.path.isfile(store_path), 'file removed')
+
+    def test_13_rollback(self):
+        self.registry.enter_test_mode(self.cr)
+        self.addCleanup(self.registry.leave_test_mode)
+        self.cr = self.registry.cursor()
+        self.addCleanup(self.cr.close)
+        self.env = odoo.api.Environment(self.cr, odoo.SUPERUSER_ID, {})
+
+        # the data needs to be unique so that no other attachment link
+        # the file so that the gc removes it
+        unique_blob = uuid.uuid4().bytes
```

Comment:
Just go get bytes from `os.urandom`, there's no reason to invoke the additional processing layers of uuid if the end-goal is to get random data (even less so as UUIDs discard 6 bits to encode the version and variant, so this is strictly less unique than just calling `os.urandom(16)`).